play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequest.scala

```diff
@@ -44,8 +45,8 @@ case class StandaloneAhcWSRequest(
   override type Self = StandaloneWSRequest
   override type Response = StandaloneWSResponse
 
-  require(client != null, "Null client!")
-  require(url != null, "Null url!")
+  require(client != null, "A [[StandaloneAhcWSClient]] is required, but it is null")
```

Comment:
"A StandaloneAhcWSClient is required, but it is null" the [[ ]] reference is for Scaladoc link, which isn't processed in the string here -- it goes into an exception.